scripts/ingestion/database/buildinfo_db_init.py

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env python
 import sqlite3
 
-conn = sqlite3.connect('/data/yellow/vineet/database/bi_multi_tables.db')
-cur = conn.cursor()
+def target_con(con):
+       conn = sqlite3.connect(con)
+       cur = conn.cursor()
+       return cur
 
+target_con('/data/yellow/vineet/database/bi_multi_tables.db')
```

Comment:
can we update this parameter to be a constant/variable that can be adjusted? something like `DATABASE_LOCATION` would be good

Reply:
Declared it as constant in constants.py